### PR TITLE
Heatmap2 tweak image quality + version update

### PR DESCRIPTION
**Pull request by tuncK**
* Update to v 3.1.3
* Increase png resolution (default is 72dpi)
* Increase colorbar size

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

### CODE REVIEW

File: tools/heatmap2/heatmap2.xml [hunk 1]
```diff
@@ -311,5 +313,11 @@ This tool employs the heatmap.2 function from the R gplots package and will gene
 Input data should have row labels in the first column and column labels. For example, the row labels (the first column) should represent gene IDs and the column labels should represent sample IDs.
     ]]></help>
     <citations>
+        <citation type="bibtex">
```
Comment (bernt-matthias) [inline tools/heatmap2/heatmap2.xml:316]: Don't think that this is necessary. The tool firm already lists that ggplot2 is used, or?

File: tools/heatmap2/heatmap2.xml [hunk 2]
```diff
@@ -110,13 +110,15 @@ ncolors <- 50
 #if $image_file_format == "pdf":
     pdf(file='$output1')
 #else:
-    png(file='$output1')
+    png(file='$output1', res=300, height=2000, width=2000)
```
Comment (bernt-matthias) [inline tools/heatmap2/heatmap2.xml:113]: What are the defaults here?

In my opinion one should not use bitmap formats if quality matters...

**Review by bernt-matthias — COMMENTED** [review]
Looks good. Just two comments and... Wondering if the margins could be smaller. In particular if the cluster tree is not shown.

File: tools/heatmap2/heatmap2.xml (hunk 1 above)
Comment (tuncK) [reply tools/heatmap2/heatmap2.xml:316]: Indeed not needed for academic plagiarism etc., but  >=1 valid citation seems to be linting requirement. I guess link to R manual would not hurt anyways.

File: tools/heatmap2/heatmap2.xml (hunk 2 above)
Comment (tuncK) [reply tools/heatmap2/heatmap2.xml:113]: 72 dpi, which blurs the gene/axis labels quite a bit.

.eps, .pdf, .svg is not very handy sometimes. In real life one often needs a quick way of inserting an image into a .docx / .pptx and send back to the collaborating PI :)

**Comment by tuncK** [comment]
> Looks good. Just two comments and... Wondering if the margins could be smaller. In particular if the cluster tree is not shown.

Sounds not very hard to do: once can add some if's into the cheetah. Will do.

File: tools/heatmap2/heatmap2.xml (hunk 2 above)
Comment (bernt-matthias) [reply tools/heatmap2/heatmap2.xml:113]: I often forget that there are still people out there using word and ppt :) anyway having a png option is perfectly fine with me.

**Comment by bernt-matthias** [comment]
> Will do.

Please, only if you think its worth the effort and you have the time for it.

**Comment by tuncK** [comment]
> Looks good. Just two comments and... Wondering if the margins could be smaller. In particular if the cluster tree is not shown.

@bernt-matthias,  fertig. The remaining whitespace , especially below, is out of abundance of caution as the user-provided strings are sometimes filenames which are much longer than car models ("HEK293_fingersCrossed_final_L001_R1.fastq.gz").

**Review by bernt-matthias — APPROVED** [review]

**Comment by tuncK** [comment]
@bernt-matthias, if you do not have any other questions/concerns, can you please merge this?

**Comment by bernt-matthias** [comment]
Sorry :) Thought I hot the automerge button. Thanks @tuncK

**Pull request merged**